docs/commands/build-docs.md

```diff
@@ -0,0 +1,73 @@
+# `build-docs`
+
+## Introduction
+
+The `build-docs` build api and Redoc into a zero-dependency HTML file.
+
+## Usage
+
+```bash
+redocly build-docs <api>
+redocly build-docs <api> --output=custom.html
+redocly build-docs <api> --options.theme.colors.primary.main=orange
+redocly build-docs <api> -t custom.hbs
+redocly build-doc <api> -t custom.hbs --templateOptions.metaDescription "Page meta description"
+```
+
+
+## Options
+
+Option | Type | Description
+-- | -- | --
+api | string | **REQUIRED.** path or URL to your to the API definition file.
+--output | string | Output file. Default value is `redoc-static.html`.
+--title | string | Page Title.
+--disableGoogleFont | boolean | Disable Google Font. Default value is `false`.
+--cdn | boolean | Use latest ReDoc version. Default value is `false`.
+--template | string | Uses custom [Handlebars](https://handlebarsjs.com/) templates to render your OpenAPI definition.
+--templateOptions | string | Adds template options you want to pass to your custom Handlebars template. To add options, use dot notation.
```

Comment:
Can you provide an example? I don't understand how this connects to dot notation and handlebars template options.

Reply:
The command is  (we have this in `Usage` section)
```bash
redocly build-docs <api> -t custom.hbs --templateOptions.metaDescription "Page meta description"
```
 And below we have `Sample Handlebars template` which contains line 
 ```
 <meta description="{{{templateOptions.metaDescription}}}">
 ```
 
That means options `templateOptions.metaDescription` from command we passed to HTML template.